src/Kafka/CreateKafkaTopic/components/CreateTopicHead.tsx

```diff
@@ -0,0 +1,69 @@
+import type { VoidFunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import {
+  PageSection,
+  PageSectionVariants,
+  Title,
+  Switch,
+  Breadcrumb,
+  BreadcrumbItem,
+} from "@patternfly/react-core";
+import { Link } from "react-router-dom";
+
+export type CreateTopicProps = {
+  showAllOptionsClicked: boolean;
```

Comment:
I don't feel like `clicked` in the property name is adding much value. Just `showAllOptions` would work.

Reply:
done